build_docs.pl

```diff
@@ -52,7 +52,7 @@ BEGIN
     'all', 'push', 'update!',    #
     'single',  'pdf',     'doc=s',           'out=s',  'toc', 'chunk=i',
     'open',    'skiplinkcheck', 'linkcheckonly', 'staging', 'procs=i',         'user=s', 'lang=s',
-    'lenient', 'verbose', 'reload_template', 'resource=s@'
+    'lenient', 'verbose', 'reload_template', 'resource=s@', 'asciidoctor'
```

Comment:
I have no idea how to write idiomatic perl. But the perl in this PR seems to work.